api/v0/observe/observe.proto

```diff
@@ -86,7 +86,7 @@ message GetAuraeDaemonLogStreamRequest {
 // TODO: not implemented
 message GetSubProcessStreamRequest {
   LogChannelType channel_type = 1;
-  int64 process_id = 2;
+  int32 process_id = 2;
```

Comment:
We have been inconsistent with the type and name for pids. Above, `Signal` we use `int64 process_id` and in cells.proto we use `int32 pid`. Do you want to make them all the same in this PR?

Reply:
i can.. outside of proto we can use libc::pid_t which is int32, so i think that's the "correct" type.